packages/anywidget/src/widget.js

```diff
@@ -309,53 +309,58 @@ class Runtime {
 	/** @type {import('solid-js').Resource<Result<AnyWidget & { url: string }>>} */
 	// @ts-expect-error - Set synchronously in constructor.
 	#widget_result;
+	/** @type {Promise<void>} */
+	model_created;
```

Comment:
Slight preference to rename this to something like `Runtime.ready`, in case there is other setup in the future that is required. e.g.,

```ts
const runtime = RUNTIMES.get(this);
await runtime?.ready;
```